x-pack/plugins/enterprise_search/public/applications/app_search/components/documents/document_detail.tsx

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { useEffect } from 'react';
+import { useActions, useValues } from 'kea';
+import { useParams } from 'react-router-dom';
+
+import {
+  EuiButton,
+  EuiPageHeader,
+  EuiPageHeaderSection,
+  EuiTitle,
+  EuiPageContentBody,
+  EuiPageContent,
+  EuiBasicTable,
+  EuiBasicTableColumn,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import { Loading } from '../../../shared/loading';
+import { SetAppSearchChrome as SetPageChrome } from '../../../shared/kibana_chrome';
+import { FlashMessages } from '../../../shared/flash_messages';
+import { ResultFieldValue } from '../result_field_value';
+
+import { DocumentDetailLogic } from './document_detail_logic';
+import { FieldDetails } from './types';
+import { DOCUMENTS_TITLE } from './constants';
+
+const DOCUMENT_DETAIL_TITLE = (documentId: string) =>
+  i18n.translate('xpack.enterpriseSearch.appSearch.documentDetail.title', {
+    defaultMessage: 'Document: {documentId}',
+    values: { documentId },
+  });
+
+const DELETE_BUTTON = i18n.translate(
+  'xpack.enterpriseSearch.appSearch.documentDetail.deleteButton',
+  {
+    defaultMessage: 'Delete',
+  }
+);
+
+interface Props {
+  engineBreadcrumb: string[];
+}
+
+export const DocumentDetail: React.FC<Props> = ({ engineBreadcrumb }) => {
+  const { dataLoading, fields } = useValues(DocumentDetailLogic);
+  const { deleteDocument, getDocumentDetails, setFields } = useActions(DocumentDetailLogic);
+
+  const { documentId } = useParams() as { documentId: string };
+
+  useEffect(() => {
+    getDocumentDetails(documentId);
+    return () => {
+      setFields([]);
+    };
+  }, []);
+
+  if (dataLoading) {
+    return <Loading />;
+  }
+
+  const deleteDocumentButton = (
+    <EuiButton
+      color="danger"
+      iconType="trash"
+      onClick={(e: React.MouseEvent) => {
+        e.preventDefault();
+        deleteDocument(documentId);
+      }}
```

Comment:
Since it's just a standard button, there shouldn't be an event that we need to prevent - we can just simplify this to

```suggestion
      onClick={() => deleteDocument(documentId)}
```

Reply:
1b0a612